CHANGELOG.md

```diff
@@ -3,10 +3,11 @@
 ## not released yet
 
 #### Features
-- Added global `--dry-run` option. It displays which command(s) will be executed without actually having a side effect. ([#90](https://github.com/peak/s5cmd/issues/90))
-- Added `--stat` option for `s5cmd` and it displays program execution statistics before the end of the program output. ([#148](https://github.com/peak/s5cmd/issues/148))
+- Added cross-region transfer support. Bucket regions are inferred, thus, supporting cross-region transfers and multiple regions in batch mode. ([#155](https://github.com/peak/s5cmd/issues/155))
 
 #### Improvements
+- Added global `--dry-run` option. It displays which command(s) will be executed without actually having a side effect. ([#90](https://github.com/peak/s5cmd/issues/90))
```

Comment:
Why we move `--dry-run` and `--stat` flags to the improvements section rather than features?

Reply:
when this pr was made, I think `--dry-run` and `--stat` were not in the master branch. During conflict resolution their place must have been shifted. fixed it. thanks for pointing it out.